pbj-core/pbj-compiler/src/main/java/com/hedera/pbj/compiler/impl/generators/TestGenerator.java

```diff
@@ -292,6 +292,14 @@ private static String generateTestMethod(final String modelClassName, final Stri
 					// check fast equals
 					dataBuffer2.resetPosition();
 					$modelClassName.PROTOBUF.fastEquals(modelObj, dataBuffer2);
```

Comment:
Think this is a bug while we are in this code, feels like it should be:
`assertTrue($modelClassName.PROTOBUF.fastEquals(modelObj, dataBuffer2));`

Reply:
Fixed.